app/src/main/java/io/homeassistant/companion/android/sensors/CarSensorManager.kt

```diff
@@ -0,0 +1,306 @@
+package io.homeassistant.companion.android.sensors
+
+import android.content.Context
+import android.content.pm.PackageManager
+import android.os.Build
+import android.util.Log
+import androidx.annotation.RequiresApi
+import androidx.car.app.hardware.common.CarValue
+import androidx.car.app.hardware.info.EnergyLevel
+import androidx.car.app.hardware.info.EvStatus
+import androidx.car.app.hardware.info.Mileage
+import androidx.car.app.hardware.info.Model
+import androidx.core.content.ContextCompat
+import androidx.lifecycle.DefaultLifecycleObserver
+import io.homeassistant.companion.android.BuildConfig
+import io.homeassistant.companion.android.common.R
+import io.homeassistant.companion.android.common.sensors.SensorManager
+import io.homeassistant.companion.android.vehicle.HaCarAppService
+
+class CarSensorManager :
+    SensorManager,
+    DefaultLifecycleObserver {
+
+    companion object {
+        internal const val TAG = "CarSM"
+
+        private val fuelLevel = SensorManager.BasicSensor(
+            "car_fuel_level",
+            "sensor",
+            R.string.basic_sensor_name_car_fuel_level,
+            R.string.sensor_description_car_fuel_level,
+            "mdi:barrel",
+            unitOfMeasurement = "%",
+            stateClass = SensorManager.STATE_CLASS_MEASUREMENT,
+            deviceClass = "battery"
+        )
+        private val batteryLevel = SensorManager.BasicSensor(
+            "car_battery_level",
```

Comment:
Same as above please drop `_level` from the ID and `Level` from the sensor name in the string